src/OpenTelemetry.Exporter.ZPages/ZPagesExporterStatsHttpServer.cs

```diff
@@ -121,9 +121,13 @@ private void WorkerThread()
                         using (var writer = new StreamWriter(output))
                         {
                             writer.WriteLine("Span Count : " + this.spanProcessor.GetSpanCount());
```

Comment:
Global count is not an interesting metric. Please start moving it under the specific span name